src/resources/MachineLearning/Models/ModelsInterfaces.ts

```diff
@@ -23,6 +23,7 @@ export interface MLModel extends MLModelInfo, ModelAttributes, GranularResource
     exportOffset?: string;
     status: ModelStatus;
     commandLineParameters?: string[];
+    extraConfig?: any;

```

Comment:
If it's still an object, wouldn't it be more interesting to do this?

```suggestion
    extraConfig?: {
        [key: string]: any;
    };
```